sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/processor/LabelsProcessor.java

```diff
@@ -0,0 +1,18 @@
+package io.opentelemetry.sdk.metrics.processor;
+
+import io.opentelemetry.api.metrics.common.Labels;
+import io.opentelemetry.context.Context;
+
+public interface LabelsProcessor {
```

Comment:
Hard to review with a lot of errors in the UI, do you mind running `./gradlew spotlessApply`?

Reply:
done